File: app/src/main/java/com/example/clubmanager/data/repo/Repo.java
package com.example.clubmanager.data.repo;

import com.example.clubmanager.data.models.Ping;
import com.example.clubmanager.data.models.Staff;

import java.util.List;

public interface Repo {
    void addPing(Ping ping);
    List<Ping> getPings();
    List<Staff> getStaff();
    List<String> getRoles();
}

File: app/src/main/java/com/example/clubmanager/data/repo/Repo.kt
package com.example.clubmanager.data.repo

import com.example.clubmanager.data.models.Ping
import com.example.clubmanager.data.models.Staff

interface Repo {
    val staff: List<Staff>
    val pings: List<Ping>

    fun addPing(p: Ping)
    fun acknowledge(pingId: String, staffId: String)
}
